documentation/modules/network-prerequisites.adoc

```diff
@@ -8,7 +8,7 @@
 
 The following prerequisites apply to all migrations:
 
-* IP addresses, VLANs, and other network configuration settings must not be changed before or during migration. The MAC addresses of the virtual machines are preserved during migration.
+* Do not change IP addresses, VLANs, and other network configuration settings before or during migration. The MAC addresses of the virtual machines (VMs) are preserved during migration.
 * The network connections between the source environment, the {virt} cluster, and the replication repository must be reliable and uninterrupted.
```

Comment:
@yaacov @mnecas  @mrnold  This is a small tweak of the previous language -- but is it missing something? Do we need to add  a phrase like "Once you define a migration plan , do not change ..." or is it good as is?

Reply:
@yaacov I changed it to "Do not change ... during a migration."